internal/db/migrations/genmigrations/generate.go

```diff
@@ -20,35 +21,77 @@ func generate(dialect string) {
 		fmt.Printf("error opening dir with dialect %s: %v\n", dialect, err)
 		os.Exit(1)
 	}
-	names, err := dir.Readdirnames(0)
+	versions, err := dir.Readdirnames(0)
 	if err != nil {
 		fmt.Printf("error reading dir names with dialect %s: %v\n", dialect, err)
 		os.Exit(1)
 	}
 	outBuf := bytes.NewBuffer(nil)
 	valuesBuf := bytes.NewBuffer(nil)
 
-	sort.Strings(names)
+	sort.Strings(versions)
 
-	for _, name := range names {
-		if !strings.HasSuffix(name, ".sql") {
-			continue
+	largestVer := 0
+	for _, ver := range versions {
+		verVal, err := strconv.Atoi(ver)
+		if err != nil {
+			if ver != "dev" {
```

Comment:
Nit, but rather than get an error on `dev` and then check if it's `dev`, I think it's nicer if you just put in a switch statement on ver, with a `dev` case and a `default` case. That way if you hit the default case and get an error from atoi, it's always known to be and treated as an error.